tests/v2/09_backends/test_local_experimental.py

```diff
@@ -216,3 +219,176 @@ async def test_indirect_shutdown_during_process_workflow(
+        assert threads[-1].daemon is True
+
+        # --- Processes
+        initial_processes = len(executor._processes)
+
+        for process in executor._processes.values():
+            assert process.is_alive() is True
+            assert process.daemon is False
+
+        # +++ SECOND SUBMIT
+        executor.submit(_sleep_and_return, 5)
+
+        # --- Threads
+        threads = threading.enumerate()
+        assert len(threads) == len(initial_threads) + 3
+
+        # --- Processes
+        assert (len(executor._processes) == initial_processes + 1) or (
+            len(executor._processes) == executor._max_workers
+        )
+        for process in executor._processes.values():
+            assert process.is_alive() is True
+            assert process.daemon is False
+
+        for _ in range(executor._max_workers + 5):
+            # There is a limit on number of processes
+            executor.submit(_sleep_and_return, 5)
+        assert len(executor._processes) == executor._max_workers
+
+        # +++ SHUTDOWN
+        with shutdown_file.open("w"):
+            pass
+        assert shutdown_file.exists()
+        time.sleep(2)
+
+        # --- Threads
+        threads = threading.enumerate()
+        # On GitHub CI there is one last thread
+        # <Connection(Thread-N, stopped daemon)>
+        assert (threads == initial_threads) or (
+            threads == initial_threads[:-1]
+        )
+
+        # --- Processes
+        assert len(executor._processes) == executor._max_workers
+        for process in executor._processes.values():
+            assert process.is_alive() is False
+            assert process.daemon is False
+
+    # --- Threads
+    threads = threading.enumerate()
+    # On GitHub CI there is one last thread
+    # <Connection(Thread-N, stopped daemon)>
+    assert (threads == initial_threads) or (threads == initial_threads[:-1])
+    # --- Processes
+    assert executor._processes is None
+
+    # `concurrent.futures.process.ProcessPoolExecutor`
+    with ProcessPoolExecutor() as executor:
+        # --- Threads
+        threads = threading.enumerate()
+        # There is no extra thread here, unlike the FractalProcessPoolExecutor
+        # On GitHub CI there is one last thread
+        # <Connection(Thread-N, stopped daemon)>
+        assert (threads == initial_threads) or (
+            threads == initial_threads[:-1]
+        )
+
+        # --- Processes
+        assert executor._processes == dict()
+
+        # +++ FIRST SUBMIT
+        executor.submit(_sleep_and_return, 2)
+
+        # --- Threads
+        threads = threading.enumerate()
+        assert (len(threads) == len(initial_threads) + 2) or (
+            len(threads) == len(initial_threads) + 1  # GitHub CI
+        )
+        assert threads[-2].name.startswith("Thread-")
+        assert isinstance(threads[-2], _ExecutorManagerThread)
+        assert threads[-2].daemon is False
+        assert threads[-2].is_alive() is True
```

Comment:
`threading.enumerate` only lists alive threads, thus I think we can skip this check.
(it's clearly interesting and nice to have, but I'd rather simplify this test a bit where possible)